k8s/agent-scaler/Dockerfile

```diff
@@ -1,31 +1,49 @@
-FROM centos:centos7
+FROM centos:7
+ARG AGENT_TAG=joshua-agent:latest
 
 # Install Python 3 and Mono
-RUN yum update all && yum install -y yum-utils && \
-    yum -y install centos-release-scl epel-release && \
-    yum -y install rh-python38-python rh-python38-python-pip gettext
-
-# kubectl
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.14.0/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /bin/
+RUN yum repolist && \
+    yum install -y \
+        centos-release-scl-rh \
+        epel-release \
+        scl-utils \
+        yum-utils && \
+    yum -y install \
+        gettext \
+        rh-python38 \
+        rh-python38-python-pip && \
+    curl -Ls https://dl.k8s.io/release/v1.20.0/bin/linux/amd64/kubectl \
+         -o /usr/local/bin/kubectl && \
+    chmod +x /usr/local/bin/kubectl
 
 # agent-scaler tools
-RUN mkdir -p /tools
-COPY agent-scaler.sh ensemble_count.py tmp/joshua_model.py /tools/
-RUN chmod +x /tools/agent-scaler.sh /tools/ensemble_count.py /tools/joshua_model.py
+COPY agent-scaler.sh /tools/
+COPY ensemble_count.py /tools/
+COPY joshua_model.py /tools/
+
+RUN chmod +x \
+    /tools/agent-scaler.sh \
+    /tools/ensemble_count.py \
+    /tools/joshua_model.py
 
 # libfdb_c.so
-RUN curl -L https://www.foundationdb.org/downloads/$(curl -L https://www.foundationdb.org/downloads/version.txt)/linux/libfdb_c_$(curl -L https://www.foundationdb.org/downloads/version.txt).so -o /lib64/libfdb_c.so
-RUN chmod +x /lib64/libfdb_c.so
+ARG FDB_VERSION="6.2.29"
```

Comment:
Potentially `curl` the version file?

Reply:
since it is an ARG, it can be passed in at build time using: `--build-arg FDB_VERSION=...` I've update the build.sh to just that.